charts/vh-video-api/values.dev.template.yaml

```diff
@@ -2,4 +2,6 @@
 java:
   image: '${IMAGE_NAME}'
   ingressHost: ${SERVICE_FQDN}
-  releaseNameOverride: ${RELEASE_NAME}
\ No newline at end of file
+  releaseNameOverride: ${RELEASE_NAME}
+  environment:
+    SERVICES__CALLBACKURI:  https://vh-video-web-pr-2171.dev.platform.hmcts.net/callback
```

Comment:
just a note from prior experience - changing this might not work, as the url is behind VPN and won't be reachable by Kinly. Let me know if it does though!